src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -47,7 +47,7 @@ internal abstract partial class AbstractReferenceFinder : IReferenceFinder
                 : state.SyntaxFacts.TryGetBindableParent(token);
             parent ??= token.Parent!;
 
-            return SymbolsMatchAsync(symbol, state, parent, cancellationToken);
+            return await SymbolsMatchAsync(symbol, state, parent, cancellationToken).ConfigureAwait(false);
```

Comment:
why was this made async/await?

Reply:
Unnecessary change, leftover from experimentations and reverts. Reverting